### ISSUE TITLE
Unable to use test helpers

### ISSUE THREAD

**Issue by derekbassett** [sir-dunxalot/ember-tooltips#148]
Following directions under Test Helpers

run `ember new tooltips-example`
then run `ember install ember-tooltips`

Copy import from example
```
import { assertTooltipVisible, assertTooltipNotVisible, assertTooltipRendered, assertTooltipNotRendered, triggerTooltipTargetEvent } from '../helpers/ember-tooltips';
```

When I run the test using `ember test --serve` I get the following error message:

```
Error: Could not find module `tooltips/tests/integration/helpers/ember-tooltips` imported from `tooltips/tests/integration/components/tooltip-example-test`
```

I could have completely missed a step but if that is the case it would be helpful to include that in the documentation about additional steps I need to do.

I can provide an example in a repository.

**Comment by derekbassett** [comment]
I was only able to get this to work with 
```import { assertTooltipVisible, assertTooltipNotVisible, assertTooltipRendered, triggerTooltipTargetEvent } from '<appname>/tests/helpers/ember-tooltips';```

This was with Ember 2.10.0

**Comment by derekbassett** [comment]
Talking with some folks it seems it wasn't working because I was building a component integration test.  The recommendation I saw was to make it explicit root-relative for the example.

**Comment by a15n** [comment]
Is this an issue that changes with different version of ember or is there an explicit approach we should add to the docs that'd work in each version and in acceptance and integration tests?

**Comment by derekbassett** [comment]
I think this issue should be resolved by changing the example to use explicit approach in the docs that work for both acceptance and component integration tests.  

So change the documentation to say something like 
```
import { assertTooltipVisible, assertTooltipNotVisible, assertTooltipRendered, triggerTooltipTargetEvent } from '<app>/tests/helpers/ember-tooltips';
```
so it will work in both acceptance and component integration tests.

**Comment by a15n** [comment]
I just confirmed that this works in both acceptance and integration tests. @sir-dunxalot when you update the test docs can you include this improvement?

**Comment by sir-dunxalot** [comment]
@a15n yes - will do.

**Comment by sir-dunxalot** [comment]
Will be fixed by #160
